src/security/ErrorDetail.cc

```diff
@@ -676,7 +676,7 @@ Security::ErrorDetail::err_descr() const
     if (!error_no)
         return "[No Error]";
 #if USE_OPENSSL
-    if (const char *err = detailEntry.descr.termedBuf())
+    if (const char *err = detailEntry.descr.c_str())
```

Comment:
[Same](https://github.com/squid-cache/squid/pull/1451/files#r1289104529) problem in this newly added change: Unlike the old condition, the new condition is always true.

Reply:
Addressed in 61fdffb